packages/flutter/test/foundation/annotations_test.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  test('test DocumentationIcon object', () {
+    const DocumentationIcon docIcon = DocumentationIcon('Test String');
+    expect(docIcon.url, contains('Test String'));
+  });
+
+  test('test Summary object', () {
```

Comment:
It looks like this test is testing the `text` property of the annotation and/or the constructor. Can we update the test name to reflect that?